packages/components/table/src/util.ts

```diff
@@ -374,6 +382,9 @@ export function createTablePopper(
   })
   on(trigger, 'mouseenter', showPopper)
   on(trigger, 'mouseleave', removePopper)
+  if (scrollContainer) {
+    on(scrollContainer, 'scroll', removePopper)
```

Comment:
**Suggestion:** I suggest we use `scrollContainer.addEventListener` since `on` has been deprecated